src/parse/get_docstring_indentation.ts

```diff
@@ -8,12 +8,12 @@ export function getDocstringIndentation(document: string, linePosition: number):
     while (currentLineNum < lines.length) {
         const line = lines[currentLineNum];
 
-        if (!blankLine(line)) {
+        if (!blankLine(line) && getIndentation(line)) {
             return getIndentation(line);
         }
 
         currentLineNum++;
     }
 
-    return "";
+    return "    ";
```

Comment:
I think we need to make the indentation more dynamic. Different people use different indentation levels and tabs instead of spaces.

Reply:
Reverted this back ✅ . I think most often when you don't want indentation on the docstring is for the module docstring, so it makes sense to have `""` be the default :)